### PR TITLE
pushing japan

### PR DESCRIPTION
**Pull request by NSPatton**

### CODE REVIEW

**Review by HPreast — CHANGES_REQUESTED** [review]
change line 12 class to "logo"

File: japan.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<head>
+    <link rel="stylesheet" href="./styles/main.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Japan</title>
+</head>
+<body>
+    <header>
+        <div class="country_image">
```
Comment (EthanMathis) [inline japan.html]: class="country_image" should actually be class="logo"

**Review by EthanMathis — APPROVED** [review]

File: japan.html [hunk 2]
```diff
@@ -9,17 +9,26 @@
 </head>
 <body>
     <header>
-        <div class="country_image">
+        <div class="Logo">
```
Comment (coltmay) [inline japan.html]: Please make "Logo" lowercase.

**Review by coltmay — CHANGES_REQUESTED** [review]

**Review by coltmay — APPROVED** [review]
Looks good!

**Review by EthanMathis — APPROVED** [review]

**Review by HPreast — APPROVED** [review]

**Pull request merged**